app/src/main/java/com/rkm/tasky/feature/loginscreen/screen/LoginScreen.kt

```diff
@@ -84,6 +85,15 @@ fun LoginScreenRoot(
             }
         }
     }
+
+    LaunchedEffect(Unit) {
+        viewModel.setEmail()
+    }
+
+    LaunchedEffect(Unit) {
+        viewModel.setPassword()
+    }
```

Comment:
Wouldn't it be possible to set this initial state in the ViewModel? Using LaunchedEffect will call this again after a screen rotation which is probably not what you want.